scripts/automation/release/version_patcher.py

```diff
@@ -0,0 +1,77 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+#pylint: disable=line-too-long
+
+import os
+import sys
+import fileinput
+
+class VersionPatcher(object):
```

Comment:
General question: is there other approaches to defining version? For example, does the `setuptools` accept environment variable?